clients/admin-ui/cypress/e2e/systems.cy.ts

```diff
@@ -545,13 +551,13 @@ describe("System management page", () => {
           cy.getByTestId("input-data_subjects").type(`anonymous{enter}`);
           cy.getByTestId("delete-btn").should("be.disabled");
           cy.getByTestId("save-btn").click();
-          cy.wait("@putSystem");
+          cy.wait("@putFidesctlSystem");
```

Comment:
haha probably! I wrote `fidesctl` because the fixture still has a system named `fidesctl system` in it. updating the fixture would probably affect a bunch of tests... but I can at least update the intercept alias :)